### ISSUE TITLE
Padronize the scroll style for the project

### ISSUE THREAD

**Issue by HeinzDev** [HeinzDev/Timothy-Project#13]
The project scrollbar changes for every browser. Must create an style and stick to it.

**Comment by HeinzDev** [comment]
should insert something like this
```
::-webkit-scrollbar {
    width: 10px;
    height: 10px;
  }

  ::-webkit-scrollbar-track {
    box-shadow: inset 0 0 5px grey; 
    border-radius: 10px;
    margin: 10px;
  }

  ::-webkit-scrollbar-thumb {
    background: gray; 
    border-radius: 5px;
  }
 
  ::-webkit-scrollbar-thumb:hover {
    background: zoms; 
  }
```

**Comment by HeinzDev** [comment]
Fixed on the last commit fbd6292
